utils/logging_test.go

```diff
@@ -0,0 +1,48 @@
+package utils
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/rs/zerolog/log"
+)
+
+func TestInitLogger(t *testing.T) {
```

Comment:
This is one of the tests that I was saying that "glue" the code in-place. Now, if we want to change our logging strategy, we'll have to change tests as well.

Leave this one for now, this critique is more for the class than for you guys :)

Reply:
Given that somehow this test is decreasing the overall codecoverage, maybe we should just remove it now? I agree that this will "glue" the code in-place

Reply:
In the end, it's your call! I'm OK with just removing it.